Yoast/javascript
# Add Czech sentence beginning assessment

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds the sentence beginnings assessment for Czech.
* [Yoast SEO Free] Implements the consecutive sentence beginnings assessment for Czech to ensure variety in a text.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

Set your website to Czech.
To create a Czech text you can use the [Czech Wikipedia](https://cs.wikipedia.org/wiki/Hlavn%C3%AD_strana)
In Yoast Seo Free:
 Create a new post and add three or more Hebrew sentences:
- that all begin with the same written-out number, like `jedno` ("one" in masc. form)
E.g. Jedno ovoce je položeno na stůl. Jedno ovoce na stole. Jedno ovoce na stole.
- that all begin with the same demonstrative pronoun, like `tento` ('this').
E.g. `Tento dokument je připraven. Tento hra skončila. Tento je místo, kde film začal`
In each of the above cases, the sentence beginning readability assessment should read: `Consecutive sentences: There is enough variety in your sentences. That's great!`.
- Write three or more Hebrew sentences that begin with the same word.
E.g. `Ahoj, jak se máš? Ahoj, mám se fajn. Ahoj světe! Ahoj každému z vás.`
The sentence beginning readability assessment should read: `Consecutive sentences: The text contains 3 consecutive sentences starting with the same word. Try to mix things up!`.

Note: It's currently not possible to test these steps in the content-analysis app

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-623
